backend/models/File_model.js

```diff
@@ -0,0 +1,41 @@
+const { ObjectId, Timestamp } = require("mongodb");
+const mongoose = require("mongoose");
+
+mongoose.connect(
+  "mongodb+srv://Mohak:yXcov2Wtwx7ikj1n@cluster0.1divz.mongodb.net/Cluster0?retryWrites=true&w=majority"
+);
+
+const File_Schema = new mongoose.Schema({
+  name: {
+    // req.body
+    type: String,
+    required: true,
+  },
+
+  userID: {
+    // JWT token
+    type: String, // ObjectId
+    required: true,
+  },
+
+  /* Time_creation : { //read and alter from what patrick sent
+        type: Date,
+        required: true
+    }, */
+
+  Category_ID: {
+    // from req.body
+    type: String,
+    required: true,
+  },
+
+  File_ID: {
```

Comment:
Amazon s3 file id should be a unique id, not a file name. It would probably make more sense to rename this to s3_id

![image](https://user-images.githubusercontent.com/29768212/152602639-881b27fc-39e4-42bd-85f8-5d0a86ea1b76.png)